setup.py

```diff
@@ -2,18 +2,18 @@
 
 setup(
     name='VacasaConnect',
-    version='4.21.2',
+    version='4.21.3',
     description='A Python 3.6+ SDK for the connect.vacasa.com API.',
     packages=['vacasa.connect'],
     url='https://github.com/Vacasa/python-vacasa-connect-sdk',
     license='MIT',
     install_requires=[
         'pendulum==2.*',
         'requests>=2.20.0',
-        'python-jose==3.0.*'
+        'python-jose==3.*'
     ],
     extras_require={
-        "cryptography": ['cryptography==2.6.*']
+        "cryptography": ['cryptography>=2.6.*']
```

Comment:
I see  "version": "==3.0" in Pipefile.lock though, so this doesn't quite match it?